### ISSUE TITLE
B-9 indifference

### ISSUE THREAD

**Issue by eoinnoble** [NaNoGenMo/2017#100]
Novel generated using Markov chains modelled off of _Star Trek: The Next Generation_ scripts.

**Comment by eoinnoble** [comment]
[Done](https://github.com/eoinnoble/b9-indifference)!

**Comment by hugovk** [comment]
Good stuff, especially the formatting!

(Tip: if you enable GitHub Pages for master in your repo settings, you can have the html served at something like https://eoinnoble.github.io/b9-indifference/output/output.html)

**Comment by eoinnoble** [comment]
Yeah, I was just about to try that, but need to fix certs on the hosting for my fonts first…

**Comment by hugovk** [comment]
Reading some of this, even if it doesn't always make sense, each character definitely speaks in the right voice. It's a bit like having Star Trek on in the background!

**Comment by eoinnoble** [comment]
It's weird how well some of them have come through, Geordi sounds especially correct to me for some reason. [Markovify](https://github.com/jsvine/markovify) won't ever suggest sentences that are a 100% match to a sentence in the source material, either.

**Comment by lizadaly** [comment]
I am so used to RNN garbage that I'd forgotten how good Markov output could be. I totally laughed at the Troi lines.
